rs/sns/testing/README.md

```diff
@@ -21,8 +21,7 @@ Since `bazel` doesn't allow to run multiple tasks in parallel, it's recommended
 targets before starting the scenario:
 ```
 bazel build //rs/pocket_ic_server:pocket-ic-server
-bazel build //rs/sns/testing:cli
-bazel build //rs/sns/testing:sns_testing_canister
+bazel build //rs/sns/testing/...
```

Comment:
Let's have a separate section in the README for how to build the required binaries, 

then just write the instructions in terms of the CLI, not in terms of `bazel run ...`

Wdyt?

Reply:
It does make sense to me if `bazel` allows to run a shell in which CLI binaries are directly available

Reply:
Do you think it makes sense to add a bazel script that will install all required binaries somewhere in `PATH` (e.g. `/usr/local/bin`)?
So that users will have to run:
```
bazel run //rs/sns/testing:install
```
This will symlink `//rs/sns/testing:sns-testing-init`, `//rs/sns/testing:sns-testing`, `//rs/sns/cli:sns` binaries to `/usr/local/bin/`

Reply:
Or alternatively, add a shell script to be used as `. scripts/setup.sh`.
This script will build all binaries and add them to `PATH`

Reply:
Something like
```
#! /usr/bin/env bash

set -e

bazel_targets=(
    "//rs/sns/testing:sns-testing-init"
    "//rs/sns/testing:sns-testing"
    "//rs/sns/cli:sns"
    "//rs/pocket_ic_server:pocket-ic-server"
)

bazel_root="$(bazel info workspace)"
bin_dir="$PWD/.sns-testing-bin"
mkdir -p "$bin_dir"

for target in "${bazel_targets[@]}"; do
    bazel build "$target"
    bazel_binary_path="$(bazel cquery "$target" \
        --output=starlark --starlark:expr="target.files.to_list()[0].path")"
    ln -sf "$bazel_root/$bazel_binary_path" "$bin_dir/"
done

export PATH="$bin_dir:$PATH"
```

Reply:
The only tricky part is running `sns-testing-init` outside of `bazel run` since it requires env variables with paths to canister WASM modules 😕

Reply:
Closing for now (discussed over slack), might get back to this part in a bit.